ocw/utils.py

```diff
@@ -335,5 +337,11 @@ def calc_climatology_monthly(dataset):
         )
         raise ValueError(error)
     else:
-        return reshape_monthly_to_annually(dataset).mean(axis=0)
+        values = reshape_monthly_to_annually(dataset).mean(axis=0)
+        
+        # A year can commence from any month
+        first_month = dataset.times[0].month
```

Comment:
Hey Mike,
Not necessarily of a particular year, but of a year. So, for e.g. if 3 years Jan -Dec of data the times should reflect 12 months Jan - Dec, if the 3years were from March - Feb then the times should reflect 12 months March - Feb